include/Minefields/Utils.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 #include <utility>
 #include "Player.h"
+#include <string>
 
 void inputBoardSize(int& height, int& width);
 int inputMines(Player& player);
 std::pair<int, int> generateRandomCoord(int width, int height);
+int inputFunc(std::string const& message);
```

Comment:
#should rename to something more meaningful - `Func` does not add value because it's already clear that this is a function. `input` is not descriptive enough, Maybe a better name would be `inputInt`, something like that.